app/src/main/java/com/teamagam/gimelgimel/app/map/view/SendGeometryActionFragment.java

```diff
@@ -48,6 +52,21 @@ public void notifyInvalidInput() {
         Snackbar.LENGTH_SHORT).show();
   }
 
+  public void pickColor(int currentColor) {
+    new SpectrumDialog.Builder(getContext()).setColors(R.array.icon_colors)
+        .setSelectedColor(currentColor)
+        .setDismissOnColorSelected(true)
+        .setOnColorSelectedListener(
+            (positiveResult, color) -> mViewModel.onColorSelected(positiveResult, color))
+        .build()
+        .show(getFragmentManager(), "ColorPicker");
```

Comment:
The color picker library requires a tag for the dialog when shown.
I replaced it with `null`.